packages/core/src/types/IImage.ts

```diff
@@ -113,6 +115,9 @@ interface IImage {
     modalityLUT?: unknown;
     voiLUT?: CPUFallbackLUT;
   };
+
+  // Information on how complete this image instance is.
+  status?: ImageQualityStatus;
```

Comment:
this should be probably imageQualityStatus and for the partial one `partialImageQualityStatus` (instead of `status` ) for clairy, since `status` might mean loading status as well